web/client/plugins/OmniBar.jsx

```diff
@@ -71,11 +74,10 @@ class OmniBar extends React.Component {
  * @memberof plugins
  */
 export default {
-    OmniBarPlugin: assign(
-        OmniBar,
-        {
-            disablePluginIf: "{state('featuregridmode') === 'EDIT' || (state('router') && state('router').includes('/geostory/shared') && state('geostorymode') !== 'edit')}"
-        }
-    ),
+    OmniBarPlugin: assign(connect((state) => (
+        { isMobile: state?.browser?.mobile }
+    ))(OmniBar), {
+        disablePluginIf: "{state('featuregridmode') === 'EDIT' || (state('router') && state('router').includes('/geostory/shared') && state('geostorymode') !== 'edit')}"
```

Comment:
Why we need to change the omnibar? This component should not be part of identify but maybe I'm missing some parts